solidity/dashboard/src/components/LiquidityRewardCard.jsx

```diff
@@ -47,7 +49,6 @@ const LiquidityRewardCard = ({
 
   return (
     <Card className={`liquidity__card tile ${wrapperClassName}`}>
-      <Icons.SantaHat className="liquidity-card__santa-hat" />
```

Comment:
Let's remove the `santa-hat.svg` file and `export { ReactComponent as SantaHat } from "../static/svg/santa-hat.svg"` in the `Icons.jsx` file.

Reply:
907c4f7